### PR TITLE
Csi 1195 improve delete delay of coins

### PR DESCRIPTION
**Pull request by HarishRocks**

### CODE REVIEW

**Comment by irshadCypherock** [comment]
Task linked: [CSI-1195 Improve delete delay of coins](https://app.clickup.com/t/37308523/CSI-1195)

File: cysync/src/pages/mainApp/sidebar/wallet/NearOneCoin.tsx [hunk 1]
```diff
@@ -232,8 +232,8 @@ const NearOneCoin: React.FC<NearOneCoinProps> = ({
     setDeleteOpen(false);
   };
   const handleDeleteConfirmation = async () => {
-    await deleteHistory(coinDetails);
     await deleteCoin(coinDetails.xpub, coinDetails.slug, walletId);
+    await deleteHistory(coinDetails);
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/NearOneCoin.tsx:236]: Can we use Promise.all here also?

File: cysync/src/pages/mainApp/sidebar/wallet/OneCoin.tsx [hunk 2]
```diff
@@ -173,8 +173,8 @@ const OneCoin: React.FC<OneCoinProps> = ({
   };
 
   const handleDeleteConfirmation = async () => {
-    await deleteHistory(coinDetails);
     await deleteCoin(coinDetails.xpub, coinDetails.slug, walletId);
+    await deleteHistory(coinDetails);
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/OneCoin.tsx:177]: Here as well.

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: cysync/src/pages/mainApp/sidebar/wallet/NearOneCoin.tsx (hunk 1 above)
Comment (HarishRocks) [reply cysync/src/pages/mainApp/sidebar/wallet/NearOneCoin.tsx:236]: No that wouldn't provide better UX

File: cysync/src/pages/mainApp/sidebar/wallet/OneCoin.tsx (hunk 2 above)
Comment (HarishRocks) [reply cysync/src/pages/mainApp/sidebar/wallet/OneCoin.tsx:177]: No that wouldn't provide better UX

**Pull request merged**